package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "eslint-config-ezhome",
-  "version": "0.6.1",
-
+  "version": "1.0.0",
```

Comment:
Please list this as `1.0.0-beta.1` version. I'd like to try it, but not sure that all necessary rules integrated yet.